docs/docs/tutorials/spring-boot-integration.md

```diff
@@ -187,6 +187,35 @@ In this case, you must explicitly specify **all** components.
 
 More details can be found [here](https://github.com/langchain4j/langchain4j-spring/blob/main/langchain4j-spring-boot-starter/src/main/java/dev/langchain4j/service/spring/AiService.java).
 
+### Listening for AI Service Registration Events
+
+After you have completed the development of the AI Service in a declarative manner, you can also listen for the
+`AiServiceRegisteredEvent` by implementing the `ApplicationListener<AiServiceRegisteredEvent>` interface.
+
+This event is triggered when AI services are registered in the Spring context, 
+allowing you to obtain information about all registered AI services and their tools at runtime. 

```

Comment:
Not related to the PR: I am just thinking out loud, is AiServiceRegisteredEvent necessary it is possibleto get all beans by class from ApplicationContext `Map<String, T> getBeansOfType(Class<T> type)`

Reply:
> Not related to the PR: I am just thinking out loud, is AiServiceRegisteredEvent necessary it is possible to get all beans by class from ApplicationContext `Map<String, T> getBeansOfType(Class<T> type)`

This method can obtain the class information of the AI service, but cannot retrieve more details, such as the description of the tools. (`List<ToolSpecification> toolSpecifications`)